docs/getting-started/installation.md

```diff
@@ -38,8 +38,17 @@ cargo install quickwit-cli
 
 ## Use the docker image
 
-Coming soon.
+If you use docker, this might be one of the quickest way to get going. 
+The following command will pull the image from [dockerhub](https://hub.docker.com/r/quickwitinc/quickwit)
```

Comment:
I own the `quickwit` namespace on DockerHub. I just gave @fulmicoton access to the account.

Reply:
Ok cool, then we can just update the needed credentials on github environment so that we can push the image to that channel.